digital-objects/omap/21-palatine-tonsil-mics/v1.0/raw/metadata.yaml

```diff
@@ -2,13 +2,13 @@ title: >-
  OMAP-21: Organ Mapping Antibody Panel (OMAP) for Multiplexed Antibody-Based Imaging of Palatine Tonsil (FFPE) with MICS on MACSima
 description: >-
  OMAP-21 was designed for MICS  (MACSima imaging cyclic staining) imaging of FFPE human tonsil samples.  Tissue fixation and antigen retrieval is described in (Spatial protein and RNA analysis on the same tissue section using MICS technology). The MACSima technology is described in detail in the following publication (MACSima imaging cyclic staining (MICS) technology reveals combinatorial target pairs for CAR T cell treatment of solid tumors). Most, but not all, antibodies in this panel are recombinant antibodies with a mutated human IgG1 constant region. The described mutation removes the Fc receptor binding capacity of human IgG1, eliminating the need for additional blocking steps and reducing non-specific binding of human antibodies on human tissues. Highly multiplexed imaging is achieved through cycles of immunolabeling with FITC, PE, and APC conjugated antibodies and photobleaching to eliminate fluorescence signal between imaging cycles. The panel contains 47 antibodies and the nuclear marker DAPI for image alignment and nuclear segmentation. This OMAP provides a spatial context for all anatomical structures and most cell types present in the human palatine tonsil. OMAP-21 follows closely OMAP-1 described for human lymph nodes (omap-1-human-lymph-node-ibex) and OMAP-10 (omap-10-palatine-tonsil-macsima). 
```

Comment:
Search and replace all double space by single spaces; here in front of (MACS...

Reply:
1 instance found/fixed.